en/docs/learn/enterprise-integration-patterns/messaging-systems/message-channels.md

```diff
@@ -1,114 +1,147 @@
 # Message Channels
 
+This page explains how you can implement a sample scenario of Message Channels using WSO2 Micro Integrator.
+
+## Introduction to Message Channels
+
 Message Channels facilitate communication between applications. A sender adds a message to a particular channel, which a receiver can read. Message Channels allow the sender and receiver to synchronize.
 
 !!! info
     For more information, see the [Message Channel](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessageChannel.html) documentation.
 
 ## Sample scenario
 
-The sample scenario depicts how a stock inventory is made from a sender application to the receiver application through a Message Channel. The Message Channel retrieves the message content from the sender, and it allows the receiver to read the content through the Message Channel. The diagram below depicts how to simulate the sample scenario using the ESB profile.
+The sample scenario depicts how a stock inventory is made from a sender application to the receiver application through a Message Channel. The Message Channel retrieves the message content from the sender, and it allows the receiver to read the content through the Message Channel. The diagram below depicts how to simulate the sample scenario.
 
 ![Message channels]({{base_path}}/assets/img/learn/enterprise-integration-patterns/messaging-systems/message-channels.png)
 
 ## Synapse configurations of the artifacts
 
-When you unzip the ZIP file you download below in Step 6 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/synapse-config` directory. For more information about these artifacts, go to WSO2 EI Documentation.
+When you unzip the ZIP file you downloaded below in Step 6 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/wso2mi/artifacts/` directory. For more information about these artifacts, go to WSO2 MI Documentation.
```

Comment:
'For more information about these artifacts, go to WSO2 MI Documentation.' Let's point to synapse artifacts page, as we are already in MI docs